### ISSUE TITLE
Allow Multiple Authors

### ISSUE THREAD

**Issue by mattelacchiato** [Reviewable/Reviewable#653]
When we do pair programming, multiple team members can be author of an pull request. Currently only the person who opens the PR is the author.
This has several downsides:
1) Me as a co-author see only a few comments if i click on the comment-box in the upper right corner. I have to expand all changes and then scroll through the code to see all comments.
2) I can't resolve comments (mark as "Done").
3) I don't receive notifications unless I involve somehow

It would be nice to be able to configure for each PR several members as authors (everybody should be able to configure this). A nice-to-have would be that every author of an commit within this PR is pre-configured as author.

**Comment by pkaminski** [comment]
Internal note for prioritization:  2 is easy to solve by changing the logic in the `review.userIsAuthor` property.  but for 1 and 3, we'd also need to add any extra authors to discussions with the right disposition, and do so retroactively if a new author appeared part-way through the review.  We'd also need to review the default "waiting on" logic, and the "unreplied discussion" logic.
